docs/vaahcms-2/backend/permissions.md

```diff
@@ -201,9 +201,34 @@ We mention some methods bellow which help you to understand the structure.
 #### Fetch Permissions
 
 ##### Method: `GET`
+##### Action: `PermissionsController@getList`
 ##### URL: `<public-url>/api/vaah/permissions/`
 
-##### Sample Request
+#### Sample Axios Request
+
+```js
+
+async getList() {
+    let options = {
+        query: vaah().clone(this.query)
+    };
+    await vaah().ajax(
+        this.ajax_url,
+        this.afterGetList,
```

Comment:
@wri-tanmoy-d001 `after` should be `suffix` not a `prefix`

Reply:
Ok sir I'll update it